app/src/renderer/components/common/NiListItemLoading.vue

```diff
@@ -0,0 +1,64 @@
+<template lang="pug">
+.li-loading
+  .ni-votes
+  .text
+    .title ████████████████████
```

Comment:
anybody seeing this black box?

Reply:
This is also a component from cosmos-forum. The black box is shown using the `dim` color and this component is used to show that the app is in the middle of loading list items.